src/System.AppContext/tests/AppContext.Switch.Validation.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Xunit;
+
+public class AppContextSwitchesTests_InputValidation
+{
+    [Fact]
+    public void NullSwitchName_EnableSwitch()
+    {
+        Assert.Throws<ArgumentNullException>(() => AppContext.SetSwitch(null, true));
```

Comment:
For Throws for argument exceptions, there's an overload that lets you specify the name of the parameter you expect to be in the exception.